Barotrauma/BarotraumaClient/ClientSource/Characters/Character.cs

```diff
@@ -425,6 +425,10 @@ void ResetInputIfPrimaryMouse(InputType inputType)
                 {
                     cam.OffsetAmount = targetOffsetAmount = item.Prefab.OffsetOnSelected * item.OffsetOnSelectedMultiplier;
                 }
+                else if (HeldItems.FirstOrDefault(item => item.GetComponent<Holdable>()?.Aimable ?? false) is Item item2 && IsKeyDown(InputType.Aim))
```

Comment:
Pattern matching should be used instead of null coalescing here. The names `item` and `item2` are not very descriptive, and they're quite misleading because they're in this case the same item.